### ISSUE TITLE
[Bug]: Ranni's quest softlocks your file completely when blaidd is not present

### ISSUE THREAD

**Issue by FMOptimist** [LukeYui/EldenRingSeamlessCoopRelease#297]
### Discord Username

FMOptimist#4211

### Describe the issue in detail

I started Ranni's questline and proceeded with speaking to iji, seluvis, and blaidd. Blaidd was not present which is normal in base game if you've completed the early area of Nokron prior to starting the quest, and just talking to iji and seluvis continues the quest as normal. However Ranni did not continue with them alone, thus the walls were stuck and even with noclipping out I had permanently lost the ability to fast travel. Ranni was unkillable in this state even with cheats, and giving myself the fingerslayer blade or cursemark of death did not progress the quest or give me the ability to fast travel again. 

### Steps to recreate the bug

Though I described it as if killing the ancestor spirit caused it, I had been playing coop with someone who had defeated morgott, radahn, renalla, and rykard on their save. I would assume killing radahn in particular in someone elses world then trying to trigger the questline in your own would cause the softlock. 

It gets more confusing however as I entered ng+ after radahn was killed in my partner's world, but it likely still conflicted anyways. 

### Version

1.2.5 (Latest)

**Comment by FMOptimist** [comment]
(June 20th for when this occured)

**Comment by Enzayer** [comment]
Just ran into it myself.

**Comment by Enzayer** [comment]
So there is a way out, but I just spent 20 min looking for the guide we used and can't find it.  I'll post what I remember and hope it works.

1.  Someone needs to create a new throwaway (TA) character.
2. The stuck person (SP) Opens their world to visitors.
3. TA joins the SP's world and touches the site of grace at the Rise, but does not rest.
4. TA exits the world, and in their world teleports up to the Rise.
5. TA opens world and SP joins.
6. Complete the Rise quest as normal.
7. TA closes their world, and now SP should be released from the tower.

Like I said I'm writing this from memory, so some things may be off.  It is possible to do it though, we managed to save 3 people this way.
